lib/sidekiq/worker_killer.rb

```diff
@@ -1,12 +1,18 @@
 require "get_process_mem"
 require "sidekiq"
-require "sidekiq/util"
```

Comment:
added in  https://github.com/klaxit/sidekiq-worker-killer/pull/14 https://github.com/klaxit/sidekiq-worker-killer/pull/12

xref 
https://gitlab.com/gitlab-org/gitlab-foss/-/blob/48d25238c386a89e8a6af218eeb290936a8f7595/lib/gitlab/sidekiq_middleware/memory_killer.rb